### PR TITLE
Plug in 2019 data

### PR DESCRIPTION
**Pull request by danielduhh**
### Description:

This switches to our new 2019 mabox style, and imports the same geojson into redux. I also updated  the logo and various references to "2018".

_**Update:**_

I removed the old data, and renamed this years files to
- polygons_2019.json
- polygon_centroids_2019.json
- friday_food_trucks_2019.json

### UI images:

Screen Shot 2019-07-10 at 11 10 16 PM

### Unit Test Approach:

Updated the data unit test

### Test Results:
- [ x ] Did you run `npm test` and did the results pass? Yes
- [ x ] Did you add unit tests that cover your changes? Yes

### CODE REVIEW

File: src/components/leftDrawer/leftDrawer.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
 .Flier {
     width: 100%;
     height: 280px;
-    background-color: #3d90b6;
+    background-color: $light-gold;
```
Comment (stephencerqueira) [inline src/components/leftDrawer/leftDrawer.scss]: Doesn't the old hex color code translate to light blue? Are we switching up the left drawer color this year?

Comment (danielduhh) [reply src/components/leftDrawer/leftDrawer.scss]: The new logo has a transparent background, so the color here will show as the fill behind.
I originally switched it to light gold, but then switched it back to light blue. Ill ask Holly what she thinks tomorrow...

Screen Shot 2019-07-10 at 9 29 04 PM

**Comment by danielduhh** [comment]
Update:

I removed the old data, and renamed this years files to

polygons_2019.json
polygon_centroids_2019.json
friday_food_trucks_2019.json

**Comment by danielduhh** [comment]
This should close #56 and #49

**Pull request merged**